libraries/abstractions/ble_hal/test/ble_test_scipts/bleAdapter.py

```diff
@@ -180,6 +180,28 @@ def isPairable():
             "Pairable",
             interface=testutils.ADAPTER_INTERFACE)
 
+    @staticmethod
+    def pair_cancelpairing():
+        isSuccessfull = True
+        try:
+            bleAdapter.getDeviceInterface(
+                bleAdapter.remoteDevice).Pair(
+                reply_handler=pairingSuccess,
+                error_handler=pairingError,
+                timeout=bleAdapter.DBUS_HANDLER_GENERIC_TIMEOUT)
+            bleAdapter.getDeviceInterface(
+                bleAdapter.remoteDevice).CancelPairing(
+                reply_handler=pairingSuccess,
```

Comment:
It would be better to separate cancel pairing to a different handler and use different event queue for it.
Success criteria for test case can be: `cancelPairingEvent.get() == True && pairingEvent.get() == False`

Reply:
Sure